src/instructlab/model/evaluate.py

```diff
@@ -191,6 +198,37 @@ def qa_pairs_to_qna_to_avg_scores(qa_pairs: list[dict]) -> dict:
     return qna_to_avg_scores
 
 
+def launch_server(
+    ctx,
+    model,
+    model_name,
+    max_workers,
+) -> tuple:
+    # pylint: disable=import-outside-toplevel
+    # First Party
+    from instructlab.model.backends import backends
+
+    if ctx.obj.config.evaluate_serve.backend == backends.VLLM:
+        ctx.obj.config.evaluate_serve.vllm.vllm_args = (
+            f"--served-model-name {model_name}"
+        )
+    if ctx.obj.config.evaluate_serve.backend == backends.LLAMA_CPP:
+        ctx.obj.config.evaluate.mt_bench.max_workers = min(max_workers, 16)
+
+    ctx.obj.config.evaluate_serve.model_path = model
+
+    backend_instance = backends.select_backend(logger, ctx.obj.config.evaluate_serve)
+    try:
+        backend_instance.run_detached(http_client(ctx.params))
```

Comment:
Could we add a comment here that the `tls` related values are the ones being passed in here?

Reply:
Done, thanks!